setup.py

```diff
@@ -8,6 +8,7 @@
 install_requires = [
     "requests",
     "posix_ipc",
+    "pyjwt",
```

Comment:
This would also need to add some other packages (`cryptography`) and I would need to confirm it works with the debian package builder...